extensions/ql-vscode/src/view/results/AlertTablePathNodeRow.tsx

```diff
@@ -86,6 +243,21 @@ export function AlertTablePathNodeRow(props: Props) {
           "[no location]"
         )}
       </td>
+      <td
+        {...selectableZebraStripe(
+          isSelected,
+          zebraIndex,
+          "vscode-codeql__taxa-cell",
+        )}
+      >
+        {userSettings.shouldShowProvenance ? (
+          <div className="vscode-codeql__taxa-cell-div">
+            {taxaLocations(step.taxa, run, handleSarifLocationClicked)}
+          </div>
+        ) : (
+          []
```

Comment:
I think `null` would be more consistent with the rest of the codebase here:

```suggestion
          null
```

Reply:
Fixed